docs/source/configuration.rst

```diff
@@ -57,6 +57,76 @@ Cloud provider configuration
     [cloud-provider]
     gcp-zone = us-east4-b
 
+.. _elb_aws_region:
+
+``AWS Region``
+^^^^^^^^^^^^^^
+
+    Name of the AWS region to use. Recommended value: ``us-east-1``.
+
+    * Default: None
+    * Values: String, any region that supports Batch, see `AWS documentation for details <https://aws.amazon.com/about-aws/global-infrastructure/regional-product-services/>`_
+
+    Also supported via the environment variable: ``ELB_GCP_REGION``.
+
+    For background information about AWS regions, please see the `AWS
+    documentation
+    <https://aws.amazon.com/about-aws/global-infrastructure/regions_az/>`_.
+
+.. code-block::
+
+    [cloud-provider]
+    aws-region = us-east-1
+
+.. _elb_aws_subnet:
+
+``AWS Subnet``
+^^^^^^^^^^^^^^
+
+    Name of the AWS subnet to use; must exist in the chosen :ref:`region
+    <elb_aws_region>`.
+
+    * Default: None
+    * Values: String
+
+.. code-block::
+
+    [cloud-provider]
+    aws-subnet = subnet-a43744d3
+
+.. _elb_aws_security_group:
+
+``AWS Security Group``
+^^^^^^^^^^^^^^^^^^^^^^
+
+    Name of the AWS security group to use; must exist in the chosen :ref:`region
+    <elb_aws_region>`.
+
+    * Default: None
+    * Values: String
+
+.. code-block::
+
+    [cloud-provider]
+    aws-security-group = sg-6cae8d08
+
+.. _elb_aws_key_pair:
+
+``AWS Key Pair``
+^^^^^^^^^^^^^^^^
+
+    This is an optional configuration setting: Name of the AWS key pair to use to login to EC2 instances; must exist in the chosen :ref:`region
```

Comment:
AWS Key Par seems to be required for 0.0.16.

Reply:
Noted, thanks. I amended the documentation.